packages/components/src/style.scss

```diff
@@ -1,3 +1,9 @@
+// Include modern WP admin color scheme variables.
```

Comment:
I think we can simply refer to this as "the default wp components color". While it _is_ part of the "modern" wp-admin color scheme, ultimately this color is about the components, and helping those components evole to a point where they can hopefully become a design system for the rest of wp-admin.

That's also why it's so important we update this, so that we are able to optimize contrast for the correct defaults.